app/models/activity.py

```diff
@@ -1,31 +1,50 @@
 from .. import db
 from sqlalchemy import Column, Integer, Boolean, DateTime
-from .user import User
-from .room_request import *
 from datetime import datetime
+from .user import User
 
 
-class  Activity(db.Model):
-    _tablename_ = "activty"
+class Activity(db.Model):
+    _tablename_ = "activity"
 
     id = db.Column(db.Integer, primary_key=True)
-    created_date = db.Column(DateTime, default= datetime.utcnow)    
+    created_at = db.Column(DateTime, default=datetime.utcnow)
     text = db.Column(db.String())
-    commenter = db.Column(db.String())
-    #forgin keys for user and 
+    
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'))
     room_request_id = db.Column(db.Integer, db.ForeignKey('room_requests.id'))
-    
-    users = db.relationship('User', backref= ('activty'))
-    room = db.relationship('RoomRequest', backref='activty')
```

Comment:
I ended up moving the `db.relationship` to the `RoomRequest` and `User` models, but in retrospect that was probably unnecessary oops 😬